samples/maps/demo/flowmap-ship-route/demo.js

```diff
@@ -49,10 +78,27 @@
                     enabled: false
                 }
             },
-            showInLegend: false
+            nullColor: '#BD935D',
+            borderColor: '#BD935D'
```

Comment:
Great idea for the demo, I have only one concern - which is the basemap color. I was thinking about something less 'punchy' and more in harmony with the arrows color, maybe something like `#8C8F8D`? Or is it too boring :)